src/main/java/org/sunbird/ratings/service/RatingService.java

```diff
@@ -0,0 +1,17 @@
+package org.sunbird.ratings.service;
+
+import org.sunbird.common.model.SBApiResponse;
+import org.sunbird.ratings.model.RequestRating;
+
+public interface RatingService {
+    public SBApiResponse getRatings(String activity_Id, String activity_Type, String userId) throws Exception; //To do
+
+    public SBApiResponse getUsers(String activity_Id, String activity_Type, String userId) throws Exception;//To do
+
+    public SBApiResponse upsertRating(String activity_Id, String activity_Type, String userId) throws Exception; //To do
+
+    public SBApiResponse getRatingSummary(String activity_Id, String activity_Type) throws Exception; //To do
+
+    public SBApiResponse upsert(RequestRating requestRating) throws Exception;
```

Comment:
We should never throw Exception which is a catchall. We need not even declare a throws clause as the custom exception thrown is a runtime exception.

Reply:
done